cpp/include/cudf/io/parquet.hpp

```diff
@@ -824,6 +833,21 @@ class parquet_writer_options_base {
    */
   void enable_utc_timestamps(bool val);
 
+  /**
+   * @brief Sets preference for writing arrow schema. Write arrow schema if set to `true`.
+   *
+   * @param val Boolean value to enable/disable writing of arrow schema.
+   */
+  void enable_write_arrow_schema(bool val);
+
+  /**
+   * @brief Sets column chunks file path to be set in the raw output metadata.
+   *
+   * @param file_paths Vector of Strings which indicates file path. Must be same size as number of
+   * data sinks in sink info
+   */
+  void set_column_chunks_file_paths(std::vector<std::string> file_paths);
+
```

Comment:
```suggestion
```

Leftover from merge?

Reply:
Thanks for catching this, Ed! Yep, leftover from merge.